src/ralph/backends/storage/fs.py

```diff
@@ -0,0 +1,139 @@
+"""FileSystem storage backend for Ralph"""
+
+import datetime
+import logging
+import sys
+from pathlib import Path
+
+from ralph.defaults import FS_STORAGE_DEFAULT_PATH
+
+from ..mixins import HistoryMixin
+from .base import BaseStorage
+
+logger = logging.getLogger(__name__)
+
+
+class FSStorage(HistoryMixin, BaseStorage):
+    """FileSystem storage backend"""
+
+    name = "fs"
+
+    def __init__(self, path=None):
+        if path is None:
+            path = FS_STORAGE_DEFAULT_PATH
+        self._path = Path(path)
+
+        if not self._path.is_dir() and not self._path.exists():
+            logger.debug(
+                "No existing FileSystemStorage, creating a new one at %s", self._path
+            )
+            self._path.mkdir(parents=True)
+
+        logger.debug("FileSystemStorage's path: %s", self._path)
+
+    def _get_filepath(self, name):
+        """Return path of the archive in the fsstorage, or throws an exception if not found"""
+        file_path = self._path.joinpath(name)
```

Comment:
Do you now the paths join operator? I love this kind of syntactic sugar:

```suggestion
        file_path = self._path / Path(name)
```

Reply:
I didn't know about it, is it specific to pathlib?
Fixed

Reply:
Yes it is.